src/world/common/NotDoesScriptExist.inc.c

```diff
@@ -0,0 +1,5 @@
+#include "common.h"
+
+static ApiStatus NotDoesScriptExist(ScriptInstance* script, s32 isInitialCall) {
```

Comment:
(This is documentation, but I've noticed it now:) this should be called `AwaitScriptComplete`